config/boot.rb

```diff
@@ -13,7 +13,6 @@
     cache_dir:            'tmp/bootsnap', # Path to your cache
     development_mode:     rails_env == "development",
     load_path_cache:      true, # optimizes the LOAD_PATH with a cache
-    autoload_paths_cache: true, # optimizes ActiveSupport autoloads with cache
```

Comment:
Bootsnap no longer supports the classic Rails class autoloader, which this option refers to.

Samson is not using the classic autoloader. Rather, it's using Zeitwerk. So removing this capability (and configuration) is of no consequence.